### PR TITLE
refactor: consistent code style

### PR DESCRIPTION
**Pull request by ivaylonikolov7**

### CODE REVIEW

File: packages/harvester/contracts/base/KeeperCompatibleHarvester.sol [hunk 1]
```diff
@@ -12,238 +12,238 @@ abstract contract KeeperCompatibleHarvester is IHarvester, Ownable {
...
     }
 
-    function _willHarvestVault(address vaultAddress_)
-        internal
-        view
-        returns (
-            bool willHarvestVault_,
-            uint256 estimatedTxCost_,
-            uint256 callRewardAmount_
-        )
```
Comment (imollov) [inline packages/harvester/contracts/base/KeeperCompatibleHarvester.sol:202]: Formatting is changed. Please run prettier format to fix it.

File: packages/harvester/contracts/base/KeeperCompatibleHarvester.sol [hunk 2]
```diff
@@ -12,238 +12,238 @@ abstract contract KeeperCompatibleHarvester is IHarvester, Ownable {
...
+    function _willHarvestVault(address _vaultAddress)
+    internal
+    view
+    returns (
+        bool willHarvestVault,
+        uint256,
+        uint256
```
Comment (imollov) [inline packages/harvester/contracts/base/KeeperCompatibleHarvester.sol:201]: Why drop the named returns?

File: packages/harvester/contracts/interfaces/IKeeperRegistry.sol [hunk 3]
```diff
@@ -58,13 +58,13 @@ interface IKeeperRegistry {
 
     function acceptOwnership() external;
 
-    function acceptPayeeship(address keeper) external;
+    function acceptPayeeship(address _keeper) external;
```
Comment (imollov) [inline packages/harvester/contracts/interfaces/IKeeperRegistry.sol:61]: Not needed in interfaces.

```suggestion
    function acceptPayeeship(address keeper) external;
```

File: packages/harvester/contracts/base/KeeperCompatibleHarvester.sol (hunk 2 above)
Comment (ivaylonikolov7) [reply packages/harvester/contracts/base/KeeperCompatibleHarvester.sol:201]: Theres already variables with the same name if we follow the same naming convention.

**Pull request merged**